### ISSUE TITLE
Tab Direction of Form

### ISSUE THREAD

**Issue by morgans3** [morgans3/NHS_Business_Intelligence_Platform_App#12]
![image](https://user-images.githubusercontent.com/63211852/169702128-486af7db-641b-4732-8cb7-9c8099a36412.png)

When navigating through the form, using the tab button makes you go in an unusual order (down instead of the next visual/logical item, which is to the right.

**Comment by luket-nhs** [comment]
I've attempted to add tab indexes to the form inputs but some angular material elements such as the mat selection list (https://material.angular.io/components/list/overview) don't seem to accept the tabindex attribute which then means that elements tab in an even worse order.

**Comment by morgans3** [comment]
Yes, so we'll unfortunately need to re-order the DOM elements so that they tab in order and are read correctly by a reader (web accessibility)
